commown/models/account_invoice.py

```diff
@@ -15,3 +15,49 @@ def _get_invoice_key_cols(self):
         except ValueError:
             pass
         return key_cols
+
+    @api.multi
+    def _multiply_investments(self, multiplier=10):
+
+        product_ids = (
+            self.env["product.template"]
+            .search(
+                [
+                    ("is_equity", "=", True),
+                    ("equity_type", "=", "invest"),
+                    "|",
+                    ("active", "=", False),
+                    ("active", "=", True),
+                ]
+            )
+            .ids
+        )
```

Comment:
Plus lisible que l'ajout dans le domaine (où tu aurais d'ailleurs pu utiliser "in" comme opérateur) :
`self.env["product.template"].with_context(active_test=False)`